traits/ctrait.py

```diff
@@ -109,6 +111,37 @@ def comparison_mode(self):
     def comparison_mode(self, value):
         ctraits.cTrait.comparison_mode.__set__(self, value)
 
+    @property
+    def property(self):
+        """ Return a tuple of callables (fget, fset, validate) for the
+        property trait."""
+        return self._get_property()
+
+    @property.setter
+    def property(self, value):
+        """ Set the fget, fset, validate callables for the property.
+
+        Parameters
+        ----------
+        value : tuple
+            Value should be the tuple of callables (fget, fset, validate).
+
+        """
+        func_arg_counts = []
+
+        for arg in value:
+            if arg is None:
+                func_arg_counts.extend([None, 0])
```

Comment:
You could refactor here to avoid repeating the `func_arg_counts.extend` call: compute `nargs` in each branch, then do `func_arg_counts.extend([arg, nargs])`.